### PR TITLE
Update Featured Content Module styling and behavior

### PR DESCRIPTION
**Pull request by Scotchester**
This PR brings in [FCM updates made in cf-layout v10.1.0](https://github.com/cfpb/capital-framework/pull/930) and updates the local code as necessary to adapt. It also makes some general video player changes.

Background in https://GHE/CFGOV/platform/issues/3087

## Changes

- Updates cf-layout and cf-core to v10.1.0
- Modifies FCM template to pass `is_fcm` option to video player template
- Modified video player template to modify close button markup and play button markup, and only show the sharing links if it's not an FCM
- Modifies video player play button Less to create play button with SVG icon, rather than background image
- Changes how FCM video appears when play button is clicked – it now animates in from the top-right

## Removals

- Removes FCM category field in Wagtail and rendering of the same in template
- Removes useless alt text from video preview images
- Removes video play button images that are now unused

## Testing

1. Pull branch
1. `yarn`
1. `gulp`
1. If you don't have a recent database dump, `./refresh-data.sh`
1. Visit a bunch of different video/FCM pages:
   a. Standard static image FCM: http://localhost:8000/practitioner-resources/your-money-your-goals/
   b. Video FCM: http://localhost:8000/prepaid-rule/
   c. Standalone video: http://localhost:8000/data-research/prepaid-accounts/issuer-instructions/
   d. Event video: localhost:8000/about-us/events/archive-past-events/fall-2018-bcfp-advisory-committee-meetings-washington-dc/

_Note that you may need to hard-refresh or disable the cache to get the updated assets._

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by Scotchester** [comment]
Amended the last commit and force-pushed to remove one more stray reference to FCM stuff in `ref.py`.

**Comment by Scotchester** [comment]
And did it one more time.

File: cfgov/unprocessed/css/video-player.less [hunk 1]
```diff
@@ -72,11 +73,19 @@
...
-    position: absolute;
-    height: @play-btn-size__px;
     width: @play-btn-size__px;
-    background: url( @play-btn-url ) no-repeat;
+    height: @play-btn-size__px;
+    border: 2px solid @gray-5;
+    border-radius: 100%;
+    position: absolute;
     z-index: 3;
+    background-color: rgba(16, 24, 32, 0.75); // CFPB Black at 80% opacity
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: Maybe include the hex value in the code comment.

File: cfgov/unprocessed/css/video-player.less [hunk 2]
```diff
@@ -72,11 +73,19 @@
...
+    color: @white;
+    font-size: @play-btn-icon-size;
+
+    &:hover {
+        background-color: rgba(0, 114, 206, 0.75); // Pacific at 80% opacity
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: Maybe include the hex value in the code comment.

File: cfgov/unprocessed/css/video-player.less [hunk 3]
```diff
@@ -85,8 +94,17 @@
     }
 
     &__bottomRight {
-        bottom: @gutter-half__em;
-        right: @gutter-half__em;
+        right: unit( @grid_gutter-width / 2 / ( @play-btn-icon-size * @base-font-size-px ), em );
+        bottom: unit( @grid_gutter-width / 2 / ( @play-btn-icon-size * @base-font-size-px ), em );
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: Calculated sizes are getting complex. I wonder if we should document how to calculate these somewhere in the CF docs.

File: cfgov/unprocessed/css/video-player.less [hunk 4]
```diff
@@ -198,8 +213,9 @@
 
 .respond-to-min( @bp-xl-min, {
     .video-player_iframe-container {
-        width: @video-width__px;
-        margin: 0 auto;
+        // width: @video-width__px;
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: Should this be removed?

File: cfgov/unprocessed/css/video-player.less [hunk 5]
```diff
@@ -208,20 +224,69 @@
...
+        height: 36px;
+        margin: 0;
+        overflow: hidden;
+    }
+
+    .video-player_close-btn {
+        border-bottom-right-radius: 0;
+        border-bottom-left-radius: 0;
+        display: block;
+        // Pushes button out of maked window when not playing.
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: `maked` > `masked`

File: cfgov/unprocessed/css/video-player.less [hunk 6]
```diff
@@ -208,20 +224,69 @@
...
+    }
+}
+
+.respond-to-min( @bp-sm-min, {
+    .o-featured-content-module.video-playing {
+        .u-flexible-container();
+        transition: 0.5s all ease-out;
+        min-height: 0;
+
+        // .video-player {
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: Can this be deleted?

File: cfgov/unprocessed/css/video-player.less [hunk 7]
```diff
@@ -208,20 +224,69 @@
...
+        opacity: 0;
+        transition-delay: 0.5s;
+        transition-duration: 0s;
+        transition-property: opacity;
+    }
+
     .o-featured-content-module_visual.video-playing {
-        position: absolute;
-        top: 0;
+        // Position the element to the right for the start of the transition
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/video-player.less]: TRIVIAL: add trailing period for consistency with other comments.

File: cfgov/unprocessed/css/video-player.less (hunk 3 above)
Comment (Scotchester) [reply cfgov/unprocessed/css/video-player.less]: That's a good idea. Maybe we should have a document of "common idioms or patterns in our Less".

**Pull request merged**